src/main/java/ru/mambetpages/controller/ArticleController.java

```diff
@@ -41,4 +46,65 @@ public GetArticleDto getArticle(@Parameter(description = "id статьи",
 
         return articleDto;
     }
+
+    @GetMapping
+    @Operation(summary = "Получение списка статей")
+    public GetArticlesDto getArticles(@RequestParam("page") int page,
+                                      @RequestParam("size") int size) {
+        GetArticlesDto articleDto = new GetArticlesDto();
+        GetArticleShortDto firstArticle = new GetArticleShortDto();
+        GetArticleShortDto secondArticle = new GetArticleShortDto();
+        AuthorDto firstAuthorDto = new AuthorDto();
+        AuthorDto secondAuthorDto = new AuthorDto();
+        List<GetArticleShortDto> articles = new ArrayList<>();
+
+        articles.add(firstArticle);
+        articles.add(secondArticle);
+
+        firstAuthorDto.setName("Ицык");
+        firstAuthorDto.setLastName("Цыпер");
+        firstAuthorDto.setId(UUID.randomUUID());
+        firstAuthorDto.setPhoto("Фото");
+
+        secondAuthorDto.setName("Игорь");
+        secondAuthorDto.setLastName("Цыба");
+        secondAuthorDto.setId(UUID.randomUUID());
+        secondAuthorDto.setPhoto("Фото");
+
+        firstArticle.setId(UUID.randomUUID());
+        firstArticle.setTitle("Название первой статьи");
+        firstArticle.setContent("Содержание первой статьи");
+        firstArticle.setImage("Изображение первой статьи");
+        firstArticle.setTags(List.of("Тег первой статьи", "Еще один тег первой статьи", "Третий тег первой статьи", "Последний тег первой статьи"));
+        firstArticle.setPublishDate(LocalDateTime.MIN);
+        firstArticle.setAuthor(firstAuthorDto);
+        firstArticle.setViews(111);
+
+        secondArticle.setId(UUID.randomUUID());
+        secondArticle.setTitle("Название второй статьи");
+        secondArticle.setContent("Содержание второй статьи");
+        secondArticle.setImage("Изображение второй статьи");
+        secondArticle.setTags(List.of("Тег второй статьи", "Еще один тег второй статьи", "Третий тег второй статьи", "Последний тег второй статьи"));
+        secondArticle.setPublishDate(LocalDateTime.MIN);
+        secondArticle.setAuthor(secondAuthorDto);
+        secondArticle.setViews(222);
+
+        articleDto.setArticles(articles);
+        articleDto.setPagesCount(76);
+
+        return articleDto;
+    }
+
+    @PostMapping
+    @Operation(summary = "Создание новой статьи")
+    public PostArticleDto postArticle(@RequestBody PostArticleDto articleDto) {
+        PostArticleDto publishArticle = new PostArticleDto();
```

Comment:
This method should return 201 status code with path to suitable GET method , without body 
see http://www.java2s.com/example/java-api/org/springframework/http/responseentity/created-1-2.html